### ISSUE TITLE
[Link] Color specs update

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#301]
## Story description

- [x] Update component documentation page with corrected color specs

## React
```
fontColor: globalTokens.mediumBlue
disabledColor: globalTokens.lightGrey
hoverFontColor: globalTokens.mediumBlue
hoverUnderlineColor: globalTokens.mediumBlue
visitedFontColor: globalTokens.purple
visitedUnderlineColor: globalTokens.purple
activeFontColor: globalTokens.black
activeUnderlineColor: globalTokens.black
focusColor: globalTokens.mediumBlue
```

## Angular **(Pending update)**

```
"--link-fontColor": globalTokens.blue
"--link-visitedFontColor": globalTokens.violet
"--link-disabledColor": globalTokens.lightGrey
"--link-hoverFontColor": globalTokens.darkBlue
"--link-underlinedBackgroundColor": globalTokens.blue
"--link-disabledUnderlinedBackgroundColor": globalTokens.lightGrey
"--link-visitedUnderlinedBackgroundColor": globalTokens.violet
```

### On-light

|   Component token                                 |   Element         |   Core token                  |   Value       |
| ------------------------------------------------- | ----------------- | ----------------------------- | ------------- |
|   `fontColor`                                         |   Label       |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `hoverFontColor`                              |   Label:hover      |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `activeFontColor`                              |   Label:active     |   `Hal-Black`                 |   `#000000`   |
|   `disabledFontColor`                           |   Label:disabled   |   `Hal-Grey-L-60`             |   `#999999`   |
|   `visitedFontColor`                             |   Label:visited    |   `Hal-Purple-S-38`       |   `#5f249f`   |
|   `hoverUnderlineColor`                     |   Underline:hover      |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `activeUnderlineColor`                     |  Underline:active     |   `Hal-Black`                 |   `#000000`   |
|   `visitedUnderlineColor`                    |  Underline:visited    |   `Hal-Purple-S-38`       |   `#5f249f`   |
|   `focusColor`                                      |   Outline:focus      |   `Hal-Blue-S-35`             |   `#0067b3`   |

### On-dark

|   Component token                                 |   Element         |   Core token                  |   Value       |
| ------------------------------------------------- | ----------------- | ----------------------------- | ------------- |
 |   `fontColorOnDark`                            |   Label       |   `Hal-Blue-L-50`             |   `#0095ff`   |
|   `hoverFontColorOnDark`                  |   Label:hover      |   `Hal-Blue-L-50`             |   `#0095ff`   |
|   `activeFontColorOnDark`                  |   Label:active     |   `Hal-White`                 |   `#ffffff`   |
|   `disabledFontColorOnDark`               |   Label:disabled   |   `Hal-Grey-S-40`             |   `#666666`   |
|   `visitedFontColorOnDark`                  |   Label:visited    |   `Hal-Purple-S-65`       |   `#a46ede`   |
|   `hoverUnderlineColorOnDark`         |   Underline:hover      |  `Hal-Blue-L-50`             |   `#0095ff`   |
|   `visitedUnderlineColorOnDark`        |  Underline:visited    |    `Hal-Purple-S-65`       |   `#a46ede`   |
|   `focusColorOnDark`                           |   Outline:focus      |   `Hal-Blue-L-50`             |   `#0095ff`   |

**Comment by mnnaveenk** [comment]
## react

fontColor: globalTokens.mediumBlue
disabledColor: globalTokens.lightGrey
hoverFontColor: globalTokens.mediumBlue
hoverUnderlineColor: globalTokens.mediumBlue
visitedFontColor: globalTokens.purple
visitedUnderlineColor: globalTokens.purple
activeFontColor: globalTokens.black
activeUnderlineColor: globalTokens.black
focusColor: globalTokens.mediumBlue

## Angular

"--link-fontColor": globalTokens.blue
"--link-visitedFontColor": globalTokens.violet
"--link-disabledColor": globalTokens.lightGrey
"--link-hoverFontColor": globalTokens.darkBlue
"--link-underlinedBackgroundColor": globalTokens.blue
"--link-disabledUnderlinedBackgroundColor": globalTokens.lightGrey
"--link-visitedUnderlinedBackgroundColor": globalTokens.violet

## Links color specs table


|   Component token                                 |   Element         |   Core token                  |   Value       |
| ------------------------------------------------- | ----------------- | ----------------------------- | ------------- |
|   `fontColor`                                     |   Link Text       |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `fontColor` (Angular)                           |   Link Text       |   `globalTokens.blue`         |   `#005FCC`   |
|   `disabledColor`                                 |   Link Disabled   |   `globalTokens.lightGrey`    |   `#D9D9D9`   |
|   `disabledUnderlinedBackgroundColor` (Angular)   |   Link            |   `globalTokens.lightGrey`    |   `#D9D9D9`   |
|   `focusColor`                                    |   Link Focus      |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `hoverFontColor`                                |   Link Hover      |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `hoverFontColor` (Angular)                      |   Link Hover      |   `globalTokens.darkBlue`     |   `#006BF6`   |
|   `hoverUnderlineColor`                           |   Link Hover      |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `visitedFontColor`                              |   Link Visited    |   `globalTokens.purple`       |   `#6f2c91`   |
|   `visitedFontColor` (Angular)                    |   Link Visited    |   `globalTokens.violet`       |   `#8800F6`   |
|   `visitedUnderlineColor`                         |   Link Visited    |   `globalTokens.purple`       |   `#6f2c91`   |
|   `visitedUnderlinedBackgroundColor` (Angular)    |   Link Visited    |   `globalTokens.violet`       |   `#8800F6`   |
|   `activeFontColor`                               |   Link Active     |   `Hal-Black`                 |   `#000000`   |
|   `activeUnderlineColor`                          |   Link Active     |   `Hal-Black`                 |   `#000000`   |
|   `underlinedBackgroundColor` (Angular)           |   Link            |   `globalTokens.blue`         |   `#005FCC`   |

**Comment by aweell** [comment]
Closed via #352

**Comment by GomezIvann** [comment]
Changes made on the Link advanced tokens:

- [ ] `iconGutter` renamed as `iconSpacing`.
